toolchain/sem_ir/type_iterator.h

```diff
@@ -136,9 +145,21 @@ class TypeIterator::Step {
   };
   // A symbolic template type value.
   struct TemplateType {};
-  // A non-type value, which can be found as a generic parameter for a type.
-  struct Value {
-    // TODO: Include the type of the value somewhere.
+  // A concrete non-type value, which can be found as a generic parameter for a
```

Comment:
Would it be fair to add blank lines between each type?

Reply:
The reason I hadn't was for grouping. I grouped all the Start* types together. Then the singleton data types together. Then I grouped the two Start* aggregator types together (maybe I should reorder these to be next to the types they aggregate - done). Then `End` and `Done` are on their own as control types. I would personally prefer it the way it is, rather than adding an extra 20 lines of whitespace in here, making it much longer to scan through, and losing the groupings.

Let me add some grouping comments to help as well.